fhir_2_ada-r4/lab/3.0.0/sturen_laboratoriumresultaten/payload/sturen_laboratoriumresultaten_2_ada.xsl

```diff
@@ -45,7 +45,10 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
 					
 					<xsl:choose>
 						<xsl:when test="count(f:Bundle/f:entry/f:resource/f:Patient) ge 2 or count(distinct-values(f:Bundle/f:entry/f:resource/(f:MedicationRequest | f:MedicationDispense | f:MedicationStatement | f:Observation)/f:subject/f:reference/@value)) ge 2">
-							<xsl:message terminate="yes">Multiple Patients and/or subject references found. Please check.</xsl:message>
+							<xsl:call-template name="util:logMessage">
+								<xsl:with-param name="level" select="$logERROR"/>
```

Comment:
This should be FATAL based on `terminate="yes"` in old situation. Furthermore add `<xsl:with-param name="terminate" select="true()"/>`.